zerver/webhooks/gitlab/tests.py

```diff
@@ -136,9 +136,31 @@ def test_create_issue_with_assignee_event_message(self) -> None:
             HTTP_X_GITLAB_EVENT="Issue Hook"
         )
 
+    def test_create_issue_with_two_assignees_event_message(self) -> None:
+        expected_subject = u"Zulip GitLab Test / Issue #2 Zulip Test Issue 2"
+        expected_message = u"Adam Birds created [Issue #2](https://gitlab.com/adambirds/zulip-gitlab-test/issues/2)(assigned to adambirds, and eeshangarg)\n\n~~~ quote\nZulip Test Issue 2\n~~~"
```

Comment:
For proper English, it should say "to adambirds and eeshangarg" without the comma.

Reply:
@timabbott: No worries, I just updated the code. In my personal life, I have become accustomed to using the serial comma in my writing, but this was buggy in the sense that the serial comma is only to be used for lists that contain 3 or more items! Thanks for catching that! :)